### DISCUSSION TITLE
The default `Squash and merge` commit will still be co-authored when the committer is the same as the PR's creator

### DISCUSSION THREAD

**Discussion by yin1999** [community/community#33311] [Pull Requests] [answered]
We have tried to use GitHub actions to create content sync PR in [mdn/translated-content](https://github.com/mdn/translated-content). In the [workflow](https://github.com/mdn/translated-content/blob/2b23ed775a0d22ecbf8b376f5f18ee8f95e11cc7/.github/workflows/sync-translated-content.yml), we just create commit (the committer is `github-actions[bot]`) and create a PR (like mdn/translated-content#8572). But the default `Squash and merge`'s commit will still be co-authored by `github-actions[bot]` itself.

![image](https://user-images.githubusercontent.com/15844309/190836878-a51ed904-9cbd-4e88-a400-dfce48160577.png)

Is there anything wrong with the workflow or others?

**Comment by pktiuk** [comment, marked as answer]
May be related with https://github.com/isaacs/github/issues/1368
